### DISCUSSION TITLE
Flood light response is slow and unreliable

### DISCUSSION THREAD

**Discussion by martinblitz** [tsightler/ring-mqtt#593] [General Discussion]
I installed a load light cam wired pro a couple of weeks ago.  I want to control the light from home assistant.  I'm running my environment in micro k8s on a rip 4 cluster.  I have deployments for HA, mosquitto and ring-mttq, among others.

The integration was simple.  But I find that turning the flood light on take 5 to 10 seconds, and turning off is unreliable.  Is this expected?

Thanks

**Comment by tsightler** [comment]
I would say it is not expected.  Cameras can be slow to display state changes, for example, if you turn on the light with the Ring app but wait to see the state change in HA, it will take up to 20 seconds, because state is only updated on 20 second polling interval.  However, the actual light state should change quickly, it's just the known state that is lagging.

I own four 1st gen Floodlight Cams, as well as several Stickup Cam Elites (no lights, but a siren), and turning the lights on/off pretty much always happens in less than 1 second.  I mean, there's not much to it, MQTT command is received, single REST API call is made to Ring.  There's pretty much no way that can take 5 seconds so, if there's a delay, it has to be added somewhere else in the chain.
